src/SprykerEco/Shared/CrefoPay/CrefoPayConfig.php

```diff
@@ -11,31 +11,114 @@
 
 class CrefoPayConfig extends AbstractBundleConfig
 {
+    /**
+     * @var string
+     */
     public const PROVIDER_NAME = 'crefoPay';
 
+    /**
+     * @var string
+     */
     public const CREFO_PAY_PAYMENT_METHOD_BILL = 'crefoPayBill';
+
+    /**
+     * @var string
+     */
     public const CREFO_PAY_PAYMENT_METHOD_CASH_ON_DELIVERY = 'crefoPayCashOnDelivery';
+
+    /**
+     * @var string
+     */
     public const CREFO_PAY_PAYMENT_METHOD_DIRECT_DEBIT = 'crefoPayDirectDebit';
+
+    /**
+     * @var string
+     */
     public const CREFO_PAY_PAYMENT_METHOD_PAY_PAL = 'crefoPayPayPal';
+
+    /**
+     * @var string
+     */
     public const CREFO_PAY_PAYMENT_METHOD_PREPAID = 'crefoPayPrepaid';
+
+    /**
+     * @var string
+     */
     public const CREFO_PAY_PAYMENT_METHOD_SOFORT = 'crefoPaySofort';
+
+    /**
+     * @var string
+     */
     public const CREFO_PAY_PAYMENT_METHOD_CREDIT_CARD = 'crefoPayCreditCard';
+
+    /**
+     * @var string
+     */
     public const CREFO_PAY_PAYMENT_METHOD_CREDIT_CARD_3D = 'crefoPayCreditCard3D';
 
+    /**
+     * @var int
+     */
     protected const USER_RISK_CLASS = 1; //Possible values: 0 -> trusted user, 1 -> default risk user, 2 -> high risk user.
```

Comment:
```suggestion
    /**
     * Possible values: 
     * 0 -> trusted user
     * 1 -> default risk user
     * 2 -> high risk user
     *
     * @var int
     */
    protected const USER_RISK_CLASS = 1;
```

I would also assume that these values exist as constants somewhere else? Something like UserRiskClass::TRUSTED_USER? If not, would it make sense to add them?